src/test/scala/org/ergoplatform/modifiers/mempool/ErgoTransactionSpec.scala

```diff
@@ -161,6 +162,29 @@ class ErgoTransactionSpec extends ErgoPropertyTest {
     }
   }
 
+  property("assets usage correctly affects transaction total cost") {
```

Comment:
I'm not sure this test really test what expected, you take 2 different transactions and compare their costs.
I propose the following scenario:
1. Generate random transaction `tx` with tokens
2. Check, that if you add 1 token to `tx.input.head` from other inputs, `tx` cost increases by `tokenAccessCost`
3. Check, that if you add 1 token to `tx.input.head` of a new type, `tx` cost increases by `2 * tokenAccessCost`
4-5. The same 2-3 for outputs